src/createNativeStackNavigator.js

```diff
@@ -1,20 +1,20 @@
-import React from 'react';
-import { StyleSheet, Platform } from 'react-native';
 import {
-  StackRouter,
+  createNavigator,
   SceneView,
   StackActions,
-  createNavigator,
-} from 'react-navigation';
-import { HeaderBackButton } from 'react-navigation-stack';
+  StackRouter,
+} from '@react-navigation/native';
```

Comment:
It should be kept as `'react-navigation'` and `'react-navigation-stack'` because it is meant for older versions of react-navigation.

Reply:
You did not change it so why resolve conversation?

Reply:
This must have reverted when i resolved conflicts with that other file and it didn't unresolve this. Will push the change again.